src/AppBundle/EventListener/CheckSQLiteEventSubscriber.php

```diff
@@ -57,4 +62,24 @@ public function handleDatabaseExceptions(ConsoleExceptionEvent $event)
             }
         }
     }
+
+    /**
+     * This method is triggered when kernel exception occurs. And checks if sqlite extension is enabled.
+     *
+     * @param GetResponseForExceptionEvent $event
+     */
+    public function onKernelException(GetResponseForExceptionEvent $event)
+    {
+        $exception = $event->getException();
+        // Since any exception thrown during a Twig template rendering is wrapped in a Twig_Error_Runtime.
+        // We must get the original exception.
+        $previousException = $exception->getPrevious();
+
+        // Driver exception may happen in controller or in twig template rendering
+        $isDriverException = ($exception instanceof DriverException || $previousException instanceof DriverException);
```

Comment:
I think we should make shure that a connection is configured to use an `sqlite` driver because it can be an error with another one (`mysql`, 'postgresql` and etc.).